gradle/build.versions.toml

```diff
@@ -39,7 +39,7 @@ maven-model = "org.apache.maven:maven-model:3.6.2"
 mockito-core = "org.mockito:mockito-core:1.9.5"
 nebula-info = "com.netflix.nebula:gradle-info-plugin:11.3.3"
 reflections = "org.reflections:reflections:0.9.12"
-shadow-plugin = "com.github.breskeby:shadow:3b035f2"
+shadow-plugin = "com.gradleup.shadow:shadow-gradle-plugin:8.3.5"
```

Comment:
Does this mean that we can remove all the jitpack repository stuff?

Reply:
yes. done.